components/fatfs/Kconfig

```diff
@@ -205,6 +205,59 @@ menu "FAT Filesystem support"
             file is opened in write-mode, the seek mechanism will automatically fallback
             to the default implementation.
 
+    choice FATFS_USE_STRFUNC_CHOICE
+        prompt "Enable string functions, f_gets(), f_putc(), f_puts() and f_printf()"
```

Comment:
Please add something along the lines of:
"These are specialized alternatives to stdio functions for working directly with FATFS without VFS."